edge/edge-core/src/test/java/io/servicecomb/edge/core/TestEdgeInvocation.java

```diff
@@ -258,6 +267,11 @@ protected void doInvoke() throws Throwable {
 
       doInvokeClassLoader = Thread.currentThread().getContextClassLoader();
     }
+  
+    @Override
+    public void sendFailResponse(Throwable throwable) {
+      doSendFailResponse = Thread.currentThread().getContextClassLoader();
```

Comment:
It's better to use a flag instead of using the classloader to check if the method is called or not.